common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
I would not recommend touching this file-- this problem type is unsupported, and I'm not sure how to actually make it work (to see whether or not the fields below should be treated as HTML).